### PR TITLE
Fixed HALBuilder to handle meta rule

### PR DESCRIPTION
**Pull request by marco-scoppetta**

### CODE REVIEW

**Comment by marco-scoppetta** [comment]
!rtg

**Comment by marco-scoppetta** [comment]
!rtg

File: grakn-engine/src/test/java/ai/grakn/engine/controller/GraqlControllerTest.java [hunk 1]
```diff
@@ -169,6 +169,14 @@ public void whenRunningQueryWithLimitEmbedded_HalResponseIsTheSameAsJava() {
     }
 
     @Test
+    public void whenMatchingRules_ResponseStatusIs200() {
+        String queryString = "match $x sub rule; get;";
```
Comment (fppt) [inline grakn-engine/src/test/java/ai/grakn/engine/controller/GraqlControllerTest.java]: Can you please change:

```
String queryString = "match $x sub rule; get;";
```

to 

```
String queryString = "match $x sub " + Schema.MetaSchema.RULE.getLabel().getValue() + "; get;";
```

I know it's SUPER long but it makes handling refactors SOOO much easier.

**Comment by marco-scoppetta** [comment]
!rtg

**Comment by burukuru** [comment]
!rtg

**Comment by marco-scoppetta** [comment]
!rtg - Build timed out (after 120 minutes). Marking the build as aborted. for the 10000000th time

**Comment by marco-scoppetta** [comment]
wow so quick to pass all tests

**Comment by marco-scoppetta** [comment]
@fppt Can you please merge this when you have time?

**Pull request merged**